[Bug]: sleek 2.0.12 - Download URL changed

### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

sleek

### Expected/Current Behaviour

The download URL changed

### Steps to Reproduce

```console
Updating 'sleek' (2.0.11 -> 2.0.12)
Downloading new version
Starting download with aria2 ...
Download: 03/23 22:14:59 [ERROR] CUID#8 - Download aborted. URI=https://github.com/ransome1/sleek/releases/download/v2.0Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/ransome1/sleek/releases/download/v2.0.1Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: da2cc8|ERR |       0B/s|C:/Users/Zottelchen/scoop/cache/sleek#2.0.12#https_github.com_ransome1_sleek_releases_download_v2.0.12_sleek-2.0.12-win-Setup.exe_dl.7z
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
```


### Possible Solution

Update the download URL & the autoupdate.

### Scoop and Buckets Version

```console
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
78090d461 (HEAD -> master, origin/master, origin/HEAD) youtube-music: Update to version 3.3.4

'games' bucket:
599db35d3 (HEAD -> master, origin/master, origin/HEAD) ryujinx: Update to version 1.1.1243

'java' bucket:
c11b58e7 (HEAD -> master, origin/master, origin/HEAD) temurin17-nightly-jre: Update to version 17.0.11-7.0.ea

'kopia' bucket:
49866d9 (HEAD -> master, origin/master, origin/HEAD) Scoop update for kopia version v0.16.0

'main' bucket:
de68f5491 (HEAD -> master, origin/master, origin/HEAD) sqlpage: Update to version 0.20.1

'nirsoft' bucket:
0382156 (HEAD -> master, origin/master, origin/HEAD) 'updated'

'nonportable' bucket:
bb0aa91 (HEAD -> master, origin/master, origin/HEAD) autohotkey-np: Update to version 2.0.12

'sysinternals' bucket:
e669cc0 (HEAD -> main, origin/main, origin/HEAD) chore(ci): switch to `pull_request_target` trigger

'versions' bucket:
d50dde4b4 (HEAD -> master, origin/master, origin/HEAD) swift-nightly: Update to version 20240323.3
```


### Scoop Config

```console
-
```


### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.4.1
PSEdition                      Core
GitCommitId                    7.4.1
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```


### Additional Softwares

_No response_

--- a/bucket/sleek.json
+++ b/bucket/sleek.json
@@ -5,7 +5,7 @@
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/ransome1/sleek/releases/download/v2.0.12/sleek-2.0.12-win-Setup.exe#/dl.7z",
+            "url": "https://github.com/ransome1/sleek/releases/download/v2.0.12/sleek-2.0.12-win-x64-Setup.exe#/dl.7z",
             "hash": "sha512:bffb2a546f1d1c3bd5838bae5b27e28fd1cac982c23733ff2ecbc1422dd4c687b383baf872ff286a52ba02db6007bebae434d8aff2b36c83fc5215c83a3d8450"
         }
     },
@@ -28,7 +28,7 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/ransome1/sleek/releases/download/v$version/sleek-$version-win-Setup.exe#/dl.7z"
+                "url": "https://github.com/ransome1/sleek/releases/download/v$version/sleek-$version-win-x64-Setup.exe#/dl.7z"
             }
         },
         "hash": {

